kv/replica_slice_test.go

```diff
@@ -110,3 +111,22 @@ func TestReplicaSetMoveToFront(t *testing.T) {
 		t.Errorf("expected order %s, got %s", exp, stores)
 	}
 }
+
+func TestReplicaSetRandomOrder(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	rs := replicaSlice(nil)
+	for i := 0; i < 5; i++ {
+		rs = append(rs, replicaInfo{ReplicaDescriptor: roachpb.ReplicaDescriptor{StoreID: roachpb.StoreID(i + 1)}})
+	}
+
+	rs.SortRandomOrder(2)
+	expNot := []roachpb.StoreID{1, 2, 4, 5, 3}
```

Comment:
Thanks for advice. My first thought was trying few outcomes as you mentioned. I changed the size of expNot to 10 to test more case. (Even though it might not sufficient)